.gitlab-ci.yml

```diff
@@ -3,9 +3,12 @@ stages:
   - deploy
 
 variables:
-  LATEST_URL:
-    value: "https://github.com/DataDog/dd-trace-php/releases/download/0.77.0/datadog-php-tracer_0.77.0_amd64.deb"
-    description: "Location where to download latest built package"
+  LATEST_SETUP:
+    value: "https://github.com/DataDog/dd-trace-php/releases/download/0.77.0/datadog-setup.php"
+    description: "Location where to download latest datadog-setup.php script"
+  LATEST_LIBRARY_x86_64_LINUX_GNU:
+    value: "https://github.com/DataDog/dd-trace-php/releases/download/0.77.0/dd-library-php-0.77.0-x86_64-linux-gnu.tar.gz"
```

Comment:
Can we please make the effort to have only a single variable here? e.g. compute LATEST_SETUP from stripping the file name and appending datadog-setup.php?

Reply:
Done!